modules/blockbuilder/blockbuilder_test.go

```diff
@@ -766,6 +766,8 @@ func TestBlockbuilder_marksOldBlocksCompacted(t *testing.T) {
 	require.NoError(t, services.StartAndAwaitRunning(ctx, b))
 	t.Cleanup(func() {
 		require.NoError(t, services.StopAndAwaitTerminated(ctx, b))
+		cancel(errors.New("test cleanup"))
+		store.Shutdown()
```

Comment:
There is a cleanup function registered at the top of the test cancelling the context. We could remove that one to favor this one